### ISSUE TITLE
Rendering Error in React due to missing "key" property

### ISSUE THREAD

**Issue by dngpng** [uxthings/popicons#1]
Got following error/warning during rendering:

```
app-index.js:34 Warning: Each child in a list should have a unique "key" prop.

Check the render method of `PopiconsSunSolid`. See https://reactjs.org/link/warning-keys for more information.
    at path
    at PopiconsSunSolid
    at button
...
```

This might has something to do with the fact the each "path" element is created in an array?

```typescript
export function PopiconsSunLine(props) {
  return (
    React.createElement("svg", { ...props, "width": "20", "height": "20", "viewBox": "0 0 20 20", "fill": "currentcolor", "xmlns": "http://www.w3.org/2000/svg" }, [
      React.createElement("path", { "d": "M10 1C9.58224 1 9.24358 1.33866 9.24358 1.75642V2.67238C9.24358 3.09014 9.58224 3.4288 10 3.4288C10.4178 3.4288 10.7564 3.09014 10.7564 2.67238V1.75642C10.7564 1.33866 10.4178 1 10 1Z", "fill": "currentcolor" }),
      React.createElement("path", { "d": "M4.70573 3.63599C4.41033 3.34059 3.93138 3.34059 3.63598 3.63599C3.34058 3.93139 3.34058 4.41033 3.63598 4.70574L4.28366 5.35341C4.57906 5.64882 5.058 5.64882 5.3534 5.35341C5.64881 5.05801 5.64881 4.57907 5.3534 4.28367L4.70573 3.63599Z", "fill": "currentcolor" }),
      ...
    ])
  )
}
```

**Comment by filipgrkinic** [comment]
@dngpng could you try running `npm i @popicons/react@latest` and let me know if you're still getting the warning?

**Comment by dngpng** [comment]
> @dngpng could you try running `npm i @popicons/react@latest` and let me know if you're still getting the warning?

Can confirm that the issue exists in `@popicons/react` version 0.0.8 and is fixed in version  0.0.9. Thx!
